### ISSUE TITLE
support for hotkey

### ISSUE THREAD

**Issue by yesilcimenahmet** [swisnl/jQuery-contextMenu#714]
Hi,

Can you add hotkey information to menu item?

Best Regards.

**Comment by bbrala** [comment]
Good point, i kinda wonder what would be the best way though to visualize that.

**Comment by yesilcimenahmet** [comment]
Thanks for agreed. I needed this urgently and found a temporary solution.

I add the hotkey property to the menu objects and after the menu has been created I manipulate all the menu DOMs by navigating.

But I would appreciate if you add a local feature.

[Image](https://imagehost.imageupload.net/2020/05/06/Adsiz8f20814a97d8e0c0.png)

**Comment by yesilcimenahmet** [comment]
Any improvement?

**Comment by Risingson** [comment]
![image](https://user-images.githubusercontent.com/5108701/106955887-25806b00-6736-11eb-9d97-c00bd439486d.png)
